### PR TITLE
chore(react-radio): adopt custom JSX pragma

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. Adopts `react-jsx-runtime` custom pragma on `react-radio`

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Avatar-mount_base.html">720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Avatar-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Button-mount_base.html">382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Button-mount.html">388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Field-mount_base.html">1301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Field-mount.html">1348</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProvider-mount_base.html">900</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProvider-mount.html">917</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProviderWithTheme-mount_base.html">117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProviderWithTheme-mount.html">122</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">99</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">98</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">110</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/InfoButton-mount_base.html">18</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/InfoButton-mount.html">23</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/MakeStyles-mount_base.html">1115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/MakeStyles-mount.html">1131</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Persona-mount_base.html">2090</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/Persona-mount.html">2023</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/SpinButton-mount_base.html">1599</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27618/perf-test-react-components/SpinButton-mount.html">1604</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                     |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                   Change |
| :-------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                               |   `29.921 kB`<br />`9.118 kB` |   `30.662 kB`<br />`9.309 kB` | `741 B` <br />`191 B`  |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                     |   `13.952 kB`<br />`5.163 kB` |   `13.984 kB`<br />`5.154 kB` |   `32 B` <br />`-9 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                         | `151.423 kB`<br />`41.266 kB` | `151.366 kB`<br />`41.281 kB` |  `-57 B` <br />`15 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>         | `133.986 kB`<br />`33.963 kB` |  `133.93 kB`<br />`33.955 kB` |  `-56 B` <br />`-8 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr> |  `81.898 kB`<br />`19.994 kB` |  `81.841 kB`<br />`20.002 kB` |   `-57 B` <br />`8 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>           |  `81.228 kB`<br />`19.805 kB` |  `81.171 kB`<br />`19.813 kB` |   `-57 B` <br />`8 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `68.69 kB`<br />`18.686 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `208.043 kB`<br />`57.817 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `38.773 kB`<br />`12.375 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.446 kB`<br />`2.185 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |   `47.03 kB`<br />`12.784 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/391878bf4d8a8210a8279a6ced8bf9a64daf4c64'>391878bf4d8a8210a8279a6ced8bf9a64daf4c64</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-radio/package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "@fluentui/react-tabster": "^9.6.5",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```
Comment (khmakoto) [inline packages/react-components/react-radio/package.json:42]: _nit:_ Can you put this above `@fluentui/react-label` in line `37` to keep alphabetical order?

**Review by khmakoto — APPROVED** [review]

**Review by behowell — APPROVED** [review]

**Pull request merged**